__tests__/app.test.js

```diff
@@ -0,0 +1,44 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const app = require("../app");
+const { seed } = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("app", () => {
+  describe("GET /api/topics", () => {
+    test("200: GET - responds with an array", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.topics).toBeInstanceOf(Array);
+        });
+    });
+
+    test("200: GET - responds with an array of topics", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+          console.log(topics);
```

Comment:
tests look good, probably dont want to be leaving console logs in there though (only place there should be a console log in your final code is in your 500 handler 👍 )